src/Button/Button.tsx

```diff
@@ -353,6 +353,8 @@ export const Button: React.FC<Props> = ({
 
                 color: getTextColor({ color, feel, theme }),
 
+                cursor: loading || disabled ? "default" : "pointer",
```

Comment:
minor: This is super minor, but I always like when these are explicit about `(loading || disabled) ? "default" : "pointer"` to distinguish from `loading || (disabled ? "default" : "pointer")`. It doesnt actually make a difference but for some reason I find it a lot easier to parse.

Reply:
Totally understand; that's actually how I wrote it but prettier had different ideas :)

Reply:
I can't self-resolve a `minor` if I'm not making the change, so I'm requesting a re-review

![image](https://user-images.githubusercontent.com/5224118/64554777-0e9fd400-d30a-11e9-9749-500f03a28961.png)

Reply:
Ah okay 😆This is fine!

Reply:
Thanks! Please 👍 and we can :shipit: